### PR TITLE
Matchup JSON formatting for fdmprinter with code conventions

### LINKED ISSUE #980: JSON code conventions
I'm starting work on getting JSON files standardized. I've based the formatting on https://github.com/Ultimaker/Meta/blob/master/code_conventions.md Please take a peek at current state here: https://code.alephobjects.com/diffusion/CT/browse/master/resources/definitions/fdmprinter.def.json
Then click 'view raw file' to see formatting even clearer.

This is using some custom settings within Eclipse/JSON Editor Plugin then hitting the Edit -> Format JSON button. Want to check that this looks sensible before going further. Additionally should I update this for all the existing printer definitions?

### PR DESCRIPTION
**Pull request by nickthetait**
Fixes #980

### CODE REVIEW

File: resources/definitions/fdmprinter.def.json [hunk 1]
```diff
@@ -265,7 +265,9 @@
                     "label": "Disallowed areas",
                     "description": "A list of polygons with areas the print head is not allowed to enter.",
                     "type": "polygons",
-                    "default_value": [],
+                    "default_value":
+                    [
+                    ],
```
Comment (nallath) [inline resources/definitions/fdmprinter.def.json]: Woa, that's in our code convention? I kinda dislike this.

Comment (Ghostkeeper) [inline resources/definitions/fdmprinter.def.json]: Code conventions say nothing about this. Either style is officially "fine".

**Comment by nickthetait** [comment]
Seems like 2.3 release is wrapping up, so I updated this PR. Anything else you need from me?

**Comment by Ghostkeeper** [comment]
Thanks!

**Pull request merged**